apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/staticelement/PageStaticController.java

```diff
@@ -0,0 +1,87 @@
+package gov.cdc.nbs.questionbank.page.content.staticelement;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.cdc.nbs.authentication.UserDetailsProvider;
+import gov.cdc.nbs.questionbank.page.content.staticelement.request.PageStaticRequests;
+import gov.cdc.nbs.questionbank.page.content.staticelement.response.AddStaticResponse;
+import lombok.extern.slf4j.Slf4j;
+
+
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/pages/{page}/static-element/")
```

Comment:
suggestion:  Would the url `/api/v1/pages/{page}/content/static` make more sense here?